driver.md

```diff
@@ -79,6 +79,7 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
  // ----------------------------------
     rule <mode> NORMAL     </mode> <k> start => #execute    ... </k>
     rule <mode> VMTESTS    </mode> <k> start => #execute    ... </k>
+    rule <mode> COVERAGE   </mode> <k> start => #execute    ... </k>
```

Comment:
We always start with `loadTx` and we can remove this rule for now.

Reply:
Removed.